cocos/particle/renderer/particle-system-renderer-data.ts

```diff
@@ -204,18 +204,23 @@ export default class ParticleSystemRenderer {
     private _particleSystem: any = null!; // ParticleSystem
```

Comment:
Could you supply the type info instead of any?

Reply:
> 
> 
> Could you supply the type info instead of any?

ParticleSystemRenderer will import ParticleSystem and ParticleSystem will import ParticleSystemRenderer too, so if we use type ParticleSystem here will cause endless import and the editor will crash.